includes/class-webmention-receiver.php

```diff
@@ -183,19 +189,23 @@ public static function default_request_handler( $data ) {
 		$commentdata = compact( 'comment_author', 'comment_author_email', 'comment_content', 'comment_parent', 'remote_source', 'remote_source_original', 'comment_approved', 'comment_type' );
 		$commentdata = array_merge( $commentdata, $data );
 
-		$commentdata = apply_filters( 'webmention_comment_data', $commentdata );
+		// Add Default Content which can be overridden using preprocess_comment
+		$commentdata = self::default_content( $commentdata );
+		$commentdata = self::default_title( $commentdata );
```

Comment:
Instead of having a filter for the content and title this can be done by preprocess_comment, so the default title should just be set, and not be a filter from its own. If we want to bring it back with a...to be deprecated notice, we can put it inside the return for the title and content functions.